database/sql/10-tables.sql

```diff
@@ -127,29 +157,10 @@ CREATE TABLE IF NOT EXISTS public.productfile (
     file VARCHAR(1200) NOT NULL,
     product_id UUID REFERENCES product (id),
     version TIMESTAMPTZ NOT NULL DEFAULT '1111-11-11T11:11:11.11Z',
+    acquirablefile_id UUID REFERENCES acquirable (id),
```

Comment:
I think the acquirablefile_id foreign key should be `REFERENCES acquirablefile (id)`

Reply:
Good catch, thanks.  Should be fixed now.